apps/communications/dialer/js/call_log.js

```diff
@@ -10,7 +10,7 @@ var CallLog = {
   _headersInterval: null,
   _empty: true,
   _dbupgrading: false,
-  _contactCache: true,
+  _contactCache: false,
```

Comment:
Gabriele, I have a doubt about what happens in teh case:  https://github.com/gabrielesvelto/gaia/blob/bug-1112577-fix-races-in-call-log-db/apps/communications/dialer/js/call_log.js#L895.  We get the notification that a contact was deleted, but we would never remove the contact link from teh CallLogDB in this case.